spec/db/models.rb

```diff
@@ -12,6 +12,8 @@ class Author < VirtualTotalTestBase
   has_many :wip_books,       -> { wip },       :class_name => "Book"
   has_and_belongs_to_many :co_books,           :class_name => "Book"
   has_many :bookmarks,                         :class_name => "Bookmark", :through => :books
+  # who thinks I am targeting them. not who am I targeting
+  has_one :primary_target, -> { all.merge(Book.order(:name)) }, :as => :focus, :class_name => "Book"
```

Comment:
would primary_book be clearer?

Reply:
or maybe `focused_on`

Reply:
I do worry about `focused_on` looking like a date field, but it does read better.

Reply:
the use case for this bug is a definite stretch.

it is not who I am focused on, but rather who I think is focused on my
and it can't be primary book because it is a polymorphic

Reply:
thanks. I went back to this and decided to introduce a new model, 'Photo'. It is a classic polymorphic model and I feel makes the code easier to understand.